docs/develop/chain/01_deployments.md

```diff
@@ -0,0 +1,35 @@
+---
+id: deployments
+title: Deployments and Services
+---
+
+KILT has two public deployments: a production one, called **Spiritnet**, and a test/dev one, called **Peregrine**.
+To learn more about how to set up a node for either environment, please check our [fullnode set up guide](02_fullnode.md).
+
+# Spiritnet
+
+Spiritnet is the production blockchain, which has been live since September 2021.
+
+|Service|Location|
+|:-:|:-:|
+|Faucet|-|
+|Polkassembly|[https://kilt.polkassembly.network](https://kilt.polkassembly.network)|
+|Public Endpoints|[KILT Protocol](https://polkadot.js.org/apps/?rpc=wss://spiritnet.kilt.io)<br/>[OnFinality](https://polkadot.js.org/apps/?rpc=wss://spiritnet.api.onfinality.io/public-ws)<br/>[Dwellir](https://polkadot.js.org/apps/?rpc=wss://kilt-rpc.dwellir.com)|
+|Sporran|[https://chrome.google.com/webstore/detail/sporran/djdnajgjcbjhhbdblkegbcgodlkkfhcl](https://chrome.google.com/webstore/detail/sporran/djdnajgjcbjhhbdblkegbcgodlkkfhcl)|
+|Stakeboard|[https://stakeboard.kilt.io](https://stakeboard.kilt.io)|
+|Subscan|[https://spiritnet.subscan.io](https://spiritnet.subscan.io)|
+|W3N service|[https://w3n.id](https://w3n.id)
+
+# Peregrine
+
+Peregrine is the public testnet, which can be used to build and test products that use the KILT blockchain, before switching to Spiritnet.
+
+|Service|Location|
+|:-:|:-:|
+|Faucet|[https://faucet.kilt.io](https://faucet.kilt.io)|
+|Polkassembly|-|
+|Public Endpoints|[KILT Protocol](https://polkadot.js.org/apps/?rpc=wss://peregrine.kilt.io/parachain-public-ws)|
+|Sporran|[https://github.com/BTE-Trusted-Entity/sporran-extension](https://github.com/BTE-Trusted-Entity/sporran-extension) (manual loading into the browser)|
+|Stakeboard|-|
+|Subscan|[https://kilt-testnet.subscan.io](https://kilt-testnet.subscan.io)|
+|W3N service|[https://main.dwo3t819w2m3x.amplifyapp.com](https://main.dwo3t819w2m3x.amplifyapp.com)
```

Comment:
Minor suggestion. 😅 

```suggestion
**Spiritnet** is the production blockchain, which has been live since September 2021.

**Peregrine** is the public testnet, which can be used to build and test products that use the KILT blockchain, before switching to Spiritnet.

|     Service      |                                                                                               Spiritnet                                                                                                |                                              Peregrine                                              |
| :--------------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------: |
|      Faucet      |                                                                                                   -                                                                                                    |                                        [Faucet][pere-faucet]                                        |
| Public Endpoints | [BOTLabs: wss://spiritnet.kilt.io][spirit-wss-kilt]<br/>[OnFinality: wss://spiritnet.api.onfinality.io/public-ws][spirit-wss-onfinality]<br/>[Dwellir: wss://kilt-rpc.dwellir.com][spirit-wss-dwellir] |                [BOTLabs: wss://peregrine.kilt.io/parachain-public-ws](pere-wss-kilt)                |
|      Wallet      |                                                                                  [Sporran](https://www.sporran.org/)                                                                                   | [GitHub](https://github.com/BTE-Trusted-Entity/sporran-extension) (manual loading into the browser) |
|    Staking UI    |                                                                                [stakeboard](https://stakeboard.kilt.io)                                                                                |                                                  -                                                  |
|  Governance UI   |                                                                                  [polkassembly][spirit-polkassembly]                                                                                   |                                                  -                                                  |
|  Chain Explorer  |                                                                                [subscan](https://spiritnet.subscan.io)                                                                                 |                             [subscan](https://kilt-testnet.subscan.io)                              |
|   W3N service    |                                                                                        [w3n.id](https://w3n.id)                                                                                        |       [https://main.dwo3t819w2m3x.amplifyapp.com](https://main.dwo3t819w2m3x.amplifyapp.com)        |


[spirit-polkassembly]: https://kilt.polkassembly.network
[spirit-wss-kilt]: https://polkadot.js.org/apps/?rpc=wss://spiritnet.kilt.io
[spirit-wss-onfinality]: https://polkadot.js.org/apps/?rpc=wss://spiritnet.api.onfinality.io/public-ws
[spirit-wss-dwellir]: https://polkadot.js.org/apps/?rpc=wss://kilt-rpc.dwellir.com

[pere-faucet]: https://faucet.kilt.io
[pere-wss-kilt]: https://polkadot.js.org/apps/?rpc=wss://peregrine.kilt.io/parachain-public-ws

```

Reply:
I merged the two tables. I think this makes sense. I changed the names in the `service` column to be more generic Wallet vs Sporran, Staking UI vs Stakeboard. I added stakekilt.com as a staking UI.

Reply:
https://github.com/KILTprotocol/docs/pull/89/commits/a8d668c2c994125f50a650ac7490aca5ec5f7493: did a bit of capitalizing on top of that. But thanks, it looks nicer now.